.env

```diff
@@ -0,0 +1 @@
+API_SECRET=abc123
```

Comment:
I forgot to mention: you shouldn't commit ENV files, since this is a key to your hashing algorithm
Make sure to delete it, commit, add it to `.gitignore`, push and then recreate it with the key